### DISCUSSION TITLE
Edit tags in the view mode

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#6461] [Ideas]
What will be a good UI widget that allows us to add/delete tags in the view template, while still able to use that dropdown menu when clicking on a tag?

Sometimes I hope I can adjust tags while viewing a tiddler. And I always found tag dropdown menu useful.

**Comment by saqimtiaz** [comment]
This wiki shows an example of mixing the core tag dropdown and tag editing interface in the view template of a tiddler: https://saqimtiaz.github.io/sq-tw/sandbox.html

**Reply by linonetwo** [reply]
Looks very nice, can it be a plugin? (Without the CSS that make tag large :P)

**Reply by saqimtiaz** [reply]
These are the two relevant tiddlers: https://saqimtiaz.github.io/sq-tw/sandbox.html#%24%3A%2Fsq%2Fui%2Fstyles%2FViewTemplate%2Ftags

You can adjust the CSS and package them as a plugin if you want to.

**Reply by kookma** [reply]
> You can adjust the CSS and package them as a plugin if you want to.

With Gatha this is a piece of cake! You can create the plugin in zero second ;-)
